client/src/components/ApplicationDialog.jsx

```diff
@@ -1,24 +1,60 @@
-import React from 'react';
+import React, { useState, useEffect, useContext } from 'react';
 import {
   Button,
   Dialog,
   DialogActions,
   DialogContent,
   DialogTitle,
   Typography,
-  CircularProgress,
+  useMediaQuery
 } from '@mui/material';
 import dayjs from 'dayjs';
 import theme from '../theme';
+import StickyHeadTable from './GenericTable';
+import careerAPI from '../services/career.api';
 import { MessageContext } from '../Contexts';
-import { useContext } from 'react';
-import applicationsAPI from '../services/applications.api';
+import PropTypes from 'prop-types';
 
+function renderField(label, value) {
+  if (label === "Level") {
+    return (
+      <Typography variant="body1" gutterBottom sx={{ color: theme.palette.text.primary }}>
+        <strong>{label}:</strong> {value === "MSc" ? "Master of Science" : value === "BSc" ? "Bachelor of Science" : ""}
+      </Typography>
+    );
+  }
+
+  return (
+    <Typography variant="body1" gutterBottom sx={{ color: theme.palette.text.primary }}>
+      <strong>{label}:</strong> {value}
+    </Typography>
+  );
+}
+
+function StudentInformation(student_id, student_email, student_title_degree, student_enrollment_year, student_nationality, submission_date, status) {
+  return (
+    <>
+      {renderField("Id", student_id)}
+      {renderField("Email", student_email)}
+      {renderField("Title Degree", student_title_degree)}
+      {renderField("Enrollment Year", student_enrollment_year)}
+      {renderField("Nationality", student_nationality)}
+      {renderField("Submission Date", dayjs(submission_date).format("DD/MM/YYYY"))}
+      {renderField("Status", status)}
+    </>
+  );
+};
```

Comment:
The user interface of this section can be improved similar to the responsive table that we implemented in AlertDialog component.